.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       # Install library
       - name: Install maybe
-        run: pip install --root-user-action=ignore --editable .
+        run: pip install --root-user-action=ignore --editable .[result]
```

Comment:
Anything inside square brackets refers to "extras" (`extras_require` added to `setup.cfg`). If you use only `.`, no extras are installed, but since we want CI to install the `result` extra (as added to `setup.cfg`), we use `.[result]` instead of a bare `.`.